packages/esm-patient-programs-app/src/programs/programs-form.workspace.tsx

```diff
@@ -93,7 +92,7 @@ const ProgramsForm: React.FC<ProgramsFormProps> = ({
       selectedProgram: currentEnrollment?.program.uuid ?? '',
       enrollmentDate: currentEnrollment?.dateEnrolled ? parseDate(currentEnrollment.dateEnrolled) : new Date(),
       completionDate: currentEnrollment?.dateCompleted ? parseDate(currentEnrollment.dateCompleted) : null,
-      enrollmentLocation: getLocationUuid() ?? '',
+      enrollmentLocation: session?.sessionLocation?.uuid ?? '',
```

Comment:
Revert to the previous. The enrolment location should be the default value if exists.